### PR TITLE
chore(starters): add AdvancedBlog

### PR DESCRIPTION
**Pull request by Bvlktech**
## Description

Adding my first Starter Blog to GatsbyJS

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -3428,3 +3428,22 @@
     - Page Transitions
     - Blog Post Template
     - Sitemap & Robots.txt generation
+- url: https://advancedblog.netlify.com/
+  repo: https://github.com/Bvlktech/AdvancedBlog
+  description: Advanced Blog Starter for GatsbyJS
+  tags:
+    - CMS: NetlifyCMS
+    - Styling: CSS-in-JS
```
Comment (amandeepmittal) [inline docs/starters.yml]: Hi,

Both CMS and Styling tags do contain any whitespace between the `:` colon and the text. Another point to note, it is `Netlify` and not `NetlifyCMS`. Please, edit these two lines into:

```md
- CMS:Netlify
- Styling:CSS-in-JS
```

For more info, check the [Categories](https://www.gatsbyjs.org/starters/?v=2) section.

**Comment by wardpeet** [comment]
I have a few problems when viewing the starter on netlify.

On a big screen I see:
image

When visiting the page/refreshing it. I always get a flicker of unstyled content. This might be something to look at.

advanced-blog-flickr

Nothing that's blocking to get merged just some feedback to improve your starter 😄.

Thank you for sharing!

File: docs/starters.yml (hunk 1 above)
Comment (Bvlktech) [reply docs/starters.yml]: Yes sorry i will fix this it was a mistake on my part

**Comment by Bvlktech** [comment]
> I have a few problems when viewing the starter on netlify.
> 
> On a big screen I see:
> image
> 
> When visiting the page/refreshing it. I always get a flicker of unstyled content. This might be something to look at.
> 
> advanced-blog-flickr
> 
> Nothing that's blocking to get merged just some feedback to improve your starter smile.
> 
> Thank you for sharing!

I will look into this i don't know why the styles are doing that as far as screen size what is the resolution for that ill make a media query for this in `./components/styles.js`

**Pull request merged**